acceptance/openstack/ces/v1/metricdata_test.go

```diff
@@ -0,0 +1,94 @@
+package v1
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ces/v1/metricdata"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestMetricData(t *testing.T) {
+	client, err := clients.NewCesV1Client()
+	th.AssertNoErr(t, err)
+
+	ecsClient, err := clients.NewComputeV1Client()
+	th.AssertNoErr(t, err)
+
+	ecs := openstack.CreateCloudServer(t, ecsClient, openstack.GetCloudServerCreateOpts(t))
+	defer openstack.DeleteCloudServer(t, ecsClient, ecs.ID)
+
+	old := time.Now().UnixMilli()
+	newOps := metricdata.MetricDataItem{
+		Metric: metricdata.MetricInfo{
+			Namespace:  "MINE.APP",
+			MetricName: "cpu_util",
+			Dimensions: []metricdata.MetricsDimension{
+				{
+					Name:  "instance_id",
+					Value: ecs.ID,
+				},
+			},
+		},
+		Ttl:         172800,
+		CollectTime: old,
+		Value:       0.09,
+		Unit:        "%",
+		Type:        "float",
+	}
+
+	err = metricdata.CreateMetricData(client, metricdata.CreateMetricDataRequest{newOps}).ExtractErr()
+	th.AssertNoErr(t, err)
+
+	batchOps := metricdata.BatchListMetricDataRequest{
+		Metrics: []metricdata.Metric{
+			{
+				Namespace:  "MINE.APP",
+				MetricName: "cpu_util",
+				Dimensions: []metricdata.MetricsDimension{
+					{
+						Name:  "instance_id",
+						Value: ecs.ID,
+					},
+				},
+			},
+		},
+		From:   old,
+		To:     time.Now().UnixMilli(),
+		Period: "1",
+		Filter: "average",
+	}
+
+	batchData, err := metricdata.BatchListMetricData(client, batchOps).Extract()
+	th.AssertNoErr(t, err)
+	th.AssertEquals(t, len(batchData), 1)
```

Comment:
It's great to test contents of the list, like checking that the data you've created is in there.
Also, `len` check will fail in case system already has some records before.